openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -2387,6 +2388,20 @@ def test_password_equals_username_validation_decision(self):
             {"username": "somephrase", "password": "somephrase"},
             {"username": "", "password": u"The password is too similar to the username."}
         )
+    
+    def test_confirm_password_matches_password(self):
+        password = 'password123'
+        self.assertValidationDecision(
+            {'password': password, 'confirm_password': password},
+            {'password': '', 'confirm_password': ''}
+        )
+
+    @ddt.data('', 'password123')
+    def test_confirm_password_doesnt_equal_password(self, confirm_password):
+        self.assertValidationDecision(
+            {'password': 'password321', 'confirm_password': confirm_password},
+            {'password': '', 'confirm_password': six.text_type(REQUIRED_FIELD_CONFIRM_PASSWORD_MSG)}
+        )
```

Comment:
Can we move this code to Edly app as well?

Reply:
These are unit tests so we should keep it with the changes